src/app/screens/Onboard/ChooseConnector/index.tsx

```diff
@@ -5,7 +5,7 @@ import lndhub from "/static/assets/icons/lndhub.png";
 import lnd from "/static/assets/icons/lnd.png";
 import eclair from "/static/assets/icons/eclair.jpg";
 import alby from "/static/assets/icons/alby.png";
-import umbrel from "/static/assets/icons/umbrel.png";
+import mynode from "/static/assets/icons/mynode.png";
```

Comment:
hmmm. seems you're deleting the umbrel connector here?

Reply:
I can add this back in, wasn't sure if it would allow me to merge and resolve the conflicts.